MANIFEST

```diff
@@ -740,6 +743,7 @@ t/80_complex.t
 t/81_babel.t
 t/82_moderncv.t
 t/83_expl3.t
+t/84_beamer.t
```

Comment:
Better to make a test and subdirectory named, perhaps, `slides` and include the new tests, along with a test case for classic LaTeX slides. (I think the schema change will cause a problem with slides)

Reply:
I wasn't aware there was something that already used the slides, I'll see about condensing the tests.